src/sentry/utils/pytest/fixtures.py

```diff
@@ -442,3 +443,8 @@ def inner(key, value):
         request.addfinalizer(lambda: ctx_mgr.__exit__(None, None, None))
 
     return inner
+
+
+@pytest.fixture
+def django_cache():
```

Comment:
nit: Should we call this `clear_django_cache` to clarify what it does? For example, when I have fixtures `redis_cache` and `django_cache`, I would expect that both are objects I can interact with.

Reply:
I've updated this fixture to yield the cache object so it's interactable now.